docs/quickstart.rst

```diff
@@ -65,6 +65,15 @@ code snippet examples of how to use basic Fairlearn functionality for those
 who are already intimately familiar with fairness in ML. The example below
 is about binary classification, but we similarly support regression.
 
+Prerequisites
+^^^^^^^^^^^^^
+
+In order to follow the quickstart guide, you need to install additional dependencies:
+
+.. code-block:: bash
+
+    pip install matplotlib
```

Comment:
My initial reaction was "But wait! You can just install fairlearn with `pip install fairlearn[customplots]`" but 

- other than me almost nobody would think of that (since I added that extra at some point)
- the name `customplots` isn't great and should probably change (just `plots`?) or we should remove that extra altogether
- Maybe we should also mention `pip install fairlearn`?

So I guess now I wonder if just the installation for fairlearn & matplotlib is best?

Reply:
> other than me almost nobody would think of that

😄 

> Maybe we should also mention `pip install fairlearn`?

To avoid any confusion I would add it as well, yes. I didn't because it's already mentioned earlier on the same page, but not logically grouped within the guide itself. Will add it and rephrase the paragraph above. 👍

Reply:
Amazing! I appreciate it, especially since a fresh set of eyes sometimes has a much clearer perspective on this 🙂